apps/system-e2e/package.json

```diff
@@ -4,19 +4,21 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "@aws-sdk/client-ses": "3.145.0",
-    "@formatjs/intl": "1.8.1",
-    "@playwright/test": "1.25.2",
-    "axios": "0.21.4",
-    "imap-simple": "5.1.0",
-    "mailparser": "3.5.0",
-    "nodemailer": "6.7.2",
-    "playwright-tesults-reporter": "1.0.1",
-    "react-intl": "5.13.1",
-    "slack-notify": "2.0.3"
-  },
-  "scripts": {},
-  "volta": {
-    "node": "16.17.0"
+    "@anev/ts-mountebank": "*",
```

Comment:
not sure this is advisable, I think we would rather stick to pinned versions

Reply:
Even with yarn.lock file, there are many ways for devs to accidentally upgrade packages without realising it or checking for regressions.

I personally think ^ (eg ^3.145.0) is a fine compromise since:

- It simplifies deduplicating packages in node_modules.
- Guarantees that deps won't upgrade to a new backwards incompatible major without an explicit package.json change.